package.json

```diff
@@ -3,9 +3,10 @@
     "docker:logs": "docker compose logs --tail 10 -f",
     "docker:up": "docker compose -f ./docker-compose.yml -f ./docker-compose.local.yml --profile mock-services up --quiet-pull --renew-anon-volumes -d",
     "docker:down": "docker compose -f ./docker-compose.yml -f ./docker-compose.local.yml --profile mock-services --profile seed down",
-    "docker:recreate": "docker compose -f ./docker-compose.yml -f ./docker-compose.local.yml --profile mock-services up -d --build --force-recreate",
+    "docker:recreate": "docker compose -f ./docker-compose.yml -f ./docker-compose.local.yml --profile mock-services up -d --build --force-recreate --remove-orphans",
     "docker:destroy": "docker compose -f ./docker-compose.yml -f ./docker-compose.local.yml down --remove-orphans -v",
     "docker:add-seeds": "docker compose --profile seed run seed-database",
+    "docker:clean-data": "docker compose --profile seed run seed-database reset_all",
```

Comment:
New command to locally reset user, team and flow data and pull a fresh set from prod.